bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryFunctor.scala

```diff
@@ -61,7 +61,7 @@ class LchQueryFunctor(queryType: LchQueryFunctor.QueryType,
 
     def makeTargetNode(target: SPTarget, env: TargetEnvironment): Option[Serializable] = {
       def targetType: Option[String] = {
-        if (env.isBasePosition(target)) Some("Base")
+        if (env.getArbitraryTargetFromAsterism eq target) Some("Base")
```

Comment:
Sure. The LTTS uses this information, it appears, but only for output purposes and to compare observation targets. It also uses coordinates, though, so this shouldn't be an issue, I wouldn't think.

I think we can label them all base, but I also don't see a good reason to not label them asterism to clarify the distinction. I think it boils down to a matter of preference over functionality.